libkbfs/init.go

```diff
@@ -376,7 +376,7 @@ func Init(ctx Context, params InitParams, keybaseServiceCn KeybaseServiceCn, onI
 
 	config.SetKeybaseService(service)
 
-	k := NewKBPKIClient(config)
+	k := NewKBPKIClient(config, log)
```

Comment:
I don't think we should pass in `log` here, as it might not have the consistent "kbfs" prefix we would get with a new call to `config.MakeLogger`.